src/net/sf/memoranda/ui/App.java

```diff
@@ -142,7 +145,29 @@ void init() {
 	public static void closeWindow() {
 		if (frame == null)
 			return;
-		frame.dispose();
+		/* 
+		 * ajcallos 2/16/2016 changed frame.dispose() to System.exit()
+		 * System.exit(0); 
+		*/
+		
+		// crdiaz3 2/17/2016 added pop-up warning on close
```

Comment:
Done; the additions there were likely influenced by the comments already present in the project.